src/panels/Menu/TopMenuBar/Menus/ViewMenu.tsx

```diff
@@ -13,11 +14,19 @@ import { TopBarMenuWrapper } from '../TopBarMenuWrapper';
 
 export function ViewMenu() {
   const { menuItems, setMenuItemVisible } = useMenuItems();
-
+  const [propertyVisibility, setPropertyVisibility] = useOptionProperty(
+    'OpenSpaceEngine.PropertyVisibility'
+  );
+  const description = usePropertyDescription('OpenSpaceEngine.PropertyVisibility');
   const dispatch = useAppDispatch();
 
+  if (!description) {
+    return <></>;
+  }
+  const { Options: data } = description.additionalData as AdditionalDataOptions;
```

Comment:
This component inslcudes more than just the user level, so please give the list a more specific name. For example
```suggestion
  const { Options: userLevelOptions } = description.additionalData as AdditionalDataOptions;
```

Also, maybe we can add a hook that gets the list of options from an option property? Then we can remove the duplicated part and `description` etc which isn't really relevant for this component

Reply:
> Also, are we sure that we want to hide this setting behind an extra click? Might make it harder for the user to see which setting they currently have on. It would be good if that was easily accessible

I think its fine that it's behind an extra click, it's not like this setting is changed a lot and I think most people are not gonna fiddle with it anyway

Reply:
I thought about adding a hook for it but it seemed neater to me to change the property itself in the engine rather than having an extra hook hiding the "ugly" code. Also making it a hook will break the pattern that we now have with the additional data